src/main/webapp/WEB-INF/index.html

```diff
@@ -0,0 +1,33 @@
+<!--
```

Comment:
I guess you don't really need copyright comment in html file.

Reply:
Agree. I wouldn't add this